<launch>
    <arg name="strands_human_help" default="true" />
	<arg name="machine" default="localhost" />
	<arg name="user" default="" />

	<machine name="$(arg machine)" address="$(arg machine)" env-loader="/opt/strands/strands_catkin_ws/devel/env.sh" user="$(arg user)" default="true"/>    

    
    <group if="$(arg strands_human_help)">
        <include file="$(find human_help_manager)/launch/strands_help_nodes.launch"/>   
    </group>
    
    <node name="monitored_nav" pkg="monitored_navigation" type="monitored_nav.py" output="screen">
    </node>


</launch>
